package.json

```diff
@@ -16,18 +16,23 @@
     "@cosmjs/stargate": "^0",
     "@fontsource/jetbrains-mono": "^4",
     "@fontsource/roboto": "^4",
+    "@popperjs/core": "^2",
     "@svgr/webpack": "^6",
+    "@tailwindcss/forms": "^0",
+    "@tailwindcss/line-clamp": "^0",
     "axios": "^0",
     "clsx": "^1",
     "compare-versions": "^4",
-    "daisyui": "^1",
+    "match-sorter": "^6",
     "next": "^12",
     "next-transpile-modules": "^9",
     "react": "^17",
     "react-datepicker": "^4.6.0",
     "react-dom": "^17",
     "react-hot-toast": "^2",
     "react-icons": "^4",
+    "react-popper": "^2",
+    "react-query": "^3",
```

Comment:
@findolor are y'all cool with adding this library to the codebase? I personally love it for managing API state, but want to make sure we're all on the same page.

Reply:
If it's any consolation, `react-datepicker` internally uses `react-popper` dependencies:

https://github.com/CosmosContracts/juno-tools/blob/6dfab36f54d69b582d77e9d49b63f0a88ff5b629/yarn.lock#L4604-L4614

Reply:
@grikomsn this comment was in reference to react-query